Library/Homebrew/dev-cmd/update-maintainers.rb

```diff
@@ -0,0 +1,64 @@
+# typed: false
+# frozen_string_literal: true
+
+require "cli/parser"
+require "utils/github"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  sig { returns(CLI::Parser) }
+  def update_maintainers_args
+    Homebrew::CLI::Parser.new do
+      description <<~EOS
+        Update the list of maintainers in the `Homebrew/brew` README.
+      EOS
+
+      named_args :none
+    end
+  end
+
+  def update_maintainers
+    update_maintainers_args.parse
+
+    # We assume that only public members wish to be included in the README
+    public_members = GitHub.public_member_usernames("Homebrew")
+
+    plc = GitHub.members_by_team("Homebrew", "plc")
+    tsc = GitHub.members_by_team("Homebrew", "tsc")
+    linux = GitHub.members_by_team("Homebrew", "linux")
+    other = GitHub.members_by_team("Homebrew", "maintainers")
+    other.except!(*[plc, tsc, linux].map(&:keys).flatten.uniq)
+
+    sentences = [plc, tsc, linux, other].map do |h|
+      h.slice!(*public_members)
+      h.each { |k, v| h[k] = "[#{v}](https://github.com/#{k})" }
+      h.values.sort.to_sentence
+    end
+
+    readme = HOMEBREW_REPOSITORY/"README.md"
+
+    content = readme.read
+    content.gsub!(/(Homebrew's \[Project Leadership Committee.*) is .*\./,
+                  "\\1 is #{sentences[0]}.")
+    content.gsub!(/(Homebrew's \[Technical Steering Committee.*) is .*\./,
+                  "\\1 is #{sentences[1]}.")
+    content.gsub!(%r{(Homebrew/brew's Linux maintainers are).*\.},
```

Comment:
```suggestion
    content.gsub!(%r{(Homebrew's Linux maintainers are).*\.},
```

Reply:
The current README text was `Homebrew/brew's Linux maintainers`, so I've pushed a commit to modify the README and apply your suggestion.